.github/workflows/buildBackupParamaterStore.yaml

```
@@ -0,0 +1,27 @@
+name: Build Backup Parameter Store
+on:
+  push:
+    paths:
+      - 'backup-parameter-store/**'
+jobs:
+  BackupParameterStore:
```

kebab case tends to be the standard for YAML properties in GHA

```suggestion
  backup-parameter-store:
```